### PR TITLE
More in depth debugging.

### PR DESCRIPTION
**Pull request by haz**
Coverage Badge

### CODE REVIEW

File: macq/extract/locm.py [hunk 1]
```diff
@@ -710,114 +754,122 @@ def _step7(
...
+                    if (
+                        sort in state_params_to_hyps
+                        and end_state in state_params_to_hyps[sort]
+                    ):
+                        for param in state_params_to_hyps[sort][end_state]:
+                            psort = None
+                            pind = None
+                            for hyp in state_params_to_hyps[sort][end_state][param]:
+                                if hyp.B == ap:
+                                    assert psort is None or psort == hyp.G_
```
Comment (e-cal) [inline macq/extract/locm.py:853]: do we still want all these assertions in the final implementation? shouldn't matter if they pass I suppose

**Review by e-cal — APPROVED** [review]
lgtm

File: macq/extract/locm.py (hunk 1 above)
Comment (haz) [reply macq/extract/locm.py:853]: Ya, my preference would be to leave them -- useful for detecting regressions.

**Comment by haz** [comment]
Any idea how to wrangle/rewrite the tests for the new format? Seems to have broken things.

**Pull request merged**